samples/core/guide/autograph_control_flow.ipynb

```diff
@@ -0,0 +1,770 @@
+    },
+    {
+      "metadata": {
+        "id": "kGXS3UWBBNoc",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "# AutoGraph: Easy control flow for graphs \n",
+        "\n",
+        "<table class=\"tfo-notebook-buttons\" align=\"left\"><td>\n",
+        "<a target=\"_blank\"  href=\"https://colab.research.google.com/github/tensorflow/models/blob/master/samples/core/guide/autograph_control_flow.ipynb\">\n",
+        "    <img src=\"https://www.tensorflow.org/images/colab_logo_32px.png\" /><span>Run in Google Colab</span></a>  \n",
+        "</td><td>\n",
+        "<a target=\"_blank\"  href=\"https://github.com/tensorflow/models/blob/master/samples/core/guide/autograph_control_flow.ipynb\"><img width=32px src=\"https://www.tensorflow.org/images/GitHub-Mark-32px.png\" /><span>View source on GitHub</span></a></td></table>"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "CydFK2CL7ZHA",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "[AutoGraph](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/contrib/autograph/README.md) helps you write complicated graph code using just plain Python -- behind the scenes, AutoGraph automatically transforms your code into the equivalent TF graph code. We support a large chunk of the Python language, which is growing. [Please see this document for what we currently support, and what we're working on](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/contrib/autograph/LIMITATIONS.md)."
+      ]
+    },
+    {
+      "metadata": {
+        "id": "mT7meGqrZTz9",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "! pip install tf-nightly\n",
+        "\n",
+        "from __future__ import division, print_function, absolute_import\n",
+        "\n",
+        "import tensorflow as tf\n",
+        "from tensorflow.contrib import autograph\n",
+        "\n",
+        "import matplotlib.pyplot as plt"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "Ry0TlspBZVvf",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "Here's a quick example of how it works. Autograph can convert functions like this:"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "aA3gOodCBkOw",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "def g(x):\n",
+        "  if x > 0:\n",
+        "    x = x * x\n",
+        "  else:\n",
+        "    x = 0.0\n",
+        "  return x"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "LICw4XQFZrhH",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "Into graph-compatible functions like this:"
```

Comment:
or graph-building, if you prefer (it feels difficult to explain what is a graph-compatible function)

Reply:
Agree with Dan for using "graph-building".
Also, should avoid using "eager mode" and "graph mode" in this doc. Prefer "eager execution" and "graph execution".

Reply:
Done.